### PR TITLE
Hb 96 skills api

### PR DESCRIPTION
**Pull request by salcot92**

### CODE REVIEW

File: app.ts [hunk 1]
```diff
@@ -1,22 +1,24 @@
...
-
 'use strict';
 
-
 // Create the server instance
 const server =  new SSServer();
 
+// Init the skills module
+const skills = new SKController(server);
+
```
Comment (paulSambolin) [inline app.ts]: move SKController and UController below PHWController (or the auth0 registration in PHWC into a higher level file).  Since Koa cascades, these routes will only be protected by auth0 if they occur after the statement in PHW

File: src/skills/skills.controller.ts [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    var data = yield parse(this);
+    data.capabilityId = this.params.capabilityId;
+
+    var response = yield SkillsService.updateCapability(data);
+
+    this.body = response.body;
+    this.status = response.status;
+  });
+
+  router.del('/:capabilityId', function * ():any {
```
Comment (westb2) [inline src/skills/skills.controller.ts]: This throws a foreign key error because of the cap_assessment table, so it won't be very useful. I think we're going to look at how we want to handle deletes/maybe restructure the database next sprint so fixing this is out of scope for this story.
